### ISSUE TITLE
White screen (on Firefox 68.3.0esr)

### ISSUE THREAD

**Issue by oschettler** [pdfernhout/storyharp-js#1]
When I use Firefox to access https://storyharp.com/v3.0/, I get a white screen. The console shows this error:

````
Error: NetworkError when attempting to fetch resource.
  Instantiating https://storyharp.com/v3.0/LogView.js
  Loading https://storyharp.com/v3.0/TSDomain.js
  Loading main.js
````

<img width="607" alt="StoryHarp_3_0" src="https://user-images.githubusercontent.com/124634/71013545-277e0980-20f1-11ea-84c6-79b9f7addd13.png">

Using Chrome works, though:

![chrome](https://user-images.githubusercontent.com/124634/71013666-55fbe480-20f1-11ea-8367-dfd2bdff34be.png)

**Comment by pdfernhout** [comment]
@oschettler StoryHarp seems to be working OK in Firefox 76.0.1 (under Ubuntu/Linux 19.10). So I am closing this. Feel free to let me know if you continue to see the issue with details of the OS and browser version.

**Comment by pdfernhout** [comment]
@oschettler More than five years later, here is another possible causes of white screens. It seems a URL with "LogView" is being filtered by uBlock Origin Lite currently and produces a somewhat similar network error unless you turn off filtering in uBlock Origin lite. It's possible earlier versions of plain uBlock Origin might have done the same if you had that installed back then. 

To help with that, I renamed that file just now to "LoggingView" to get around uBlock Origin lite's false positive. (A different approach might have been to switch to using a JavaScript bundler to hide the individual files.) 

I checked in that change and some other changes to the repository -- including updating various packages to later versions. 

I also updated demo site with that change here: https://storyharp.com/v3.0/
